v2/gcs-to-sourcedb/src/test/java/com/google/cloud/teleport/v2/templates/GCSToSourceDbWithoutReaderIT.java

```diff
@@ -123,14 +135,73 @@ private void assertRowInMySQL() throws InterruptedException {
     long rowCount = 0;
     for (int i = 0; rowCount != 1 && i < 60; ++i) {
       rowCount = jdbcResourceManager.getRowCount(TABLE);
-      LOG.info("Row count = {}, Waiting for 30s if row count not = 1", rowCount);
+      LOG.info("Row count = {}, Waiting for 10s if row count not = 1", rowCount);
       Thread.sleep(10000);
     }
     assertThat(rowCount).isEqualTo(1);
     List<Map<String, Object>> rows = jdbcResourceManager.readTable(TABLE);
     assertThat(rows).hasSize(1);
     assertThat(rows.get(0).get("id")).isEqualTo(1);
     assertThat(rows.get(0).get("name")).isEqualTo("FF");
+
+    for (int i = 0; rowCount != 2 && i < 10; ++i) {
+      rowCount = jdbcResourceManager.getRowCount(TABLE2);
+      LOG.info("Row count = {}, Waiting for 10s if row count not = 2", rowCount);
+      Thread.sleep(10000);
+    }
+    assertThat(rowCount).isEqualTo(2);
+    rows = jdbcResourceManager.readTable(TABLE2);
+    assertThat(rows).hasSize(2);
+    String sortColumn = "varchar_column";
+    List<Map<String, Object>> sortedRows =
```

Comment:
You can give a select statement that orders it by varchar_column rather than a sort here